dask_sql/physical/rex/core/literal.py

```diff
@@ -161,6 +161,9 @@ def convert(
         elif literal_type == "IntervalDayTime":
             literal_type = SqlTypeName.INTERVAL_DAY
             literal_value = rex.getIntervalDayTimeValue()
+        elif literal_type == "IntervalMonthDayNano":
+            literal_type = SqlTypeName.INTERVAL_MONTH_DAY_NANOSECOND
+            literal_value = rex.getIntervalMonthDayNanoValue()
```

Comment:
Same question around the `IntervalDayTime` case being reached as above

Reply:
It is possible so I think we leave it. Our tests just don't hit it.